priv/templates/extended_bin_windows

```diff
@@ -332,7 +332,7 @@ set description=Erlang node %node_name%%hostname% in %rootdir%
 :attach
 @set boot=-boot "%clean_boot_script%" -boot_var RELEASE_DIR "%release_root_dir%"
 @start "%node_name% attach" %werl% %boot% ^
-       -remsh %node_name%%hostname% %node_type% console -setcookie %cookie%
+       -remsh %node_name%%hostname% %node_type% console%node_name%%random% -setcookie %cookie%
```

Comment:
for readability it might make sense to set the name to `console_%node_name%_%random%` with the extra underscores.

Reply:
Good point.

@c-bik can you make this change? Then I'll merge.

Reply:
Done!